packages/react-components/react-tabster/package.json

```diff
@@ -36,7 +36,7 @@
     "@fluentui/react-theme": "^9.0.0",
     "@fluentui/react-utilities": "^9.0.2",
     "keyborg": "^1.1.0",
-    "tabster": "^1.4.2",
+    "tabster": "^2.1.0",
```

Comment:
are there any breaking changes that might affect users (bumping to major) ?

Reply:
not to users, all encapsulated with `react-tabster`